src/pages/Build.vue

```diff
@@ -271,6 +296,16 @@ export default defineComponent({
       }
       return rebuilt
     },
+    allowDeleteBuild () {
+      let allow_delete = true
+      for (let task of this.build.tasks) {
+        if (task.status < BuildStatus.COMPLETED) {
```

Comment:
If you delete the build when it built, but not started to test it could be in the queue for testing. And I don't what happens